docs/site/_includes/getting_started/global/partials/STEP_INSTALL_SCHEMA_RU.md

```diff
@@ -1,27 +1,60 @@
+{% endif %}
+
 ## Процесс установки
 
 Вам потребуются:
 1. **Персональный компьютер**.
  
-   Компьютер, с которого будет производиться установка. Он нужен только для запуска инсталлятора Deckhouse, и не будет частью кластера. 
+   Компьютер, с которого будет производиться установка. {% unless page.platform_type == "existing" %}Он нужен только для запуска инсталлятора Deckhouse, и не будет частью кластера.{% endunless %}
    
    Требования:
    - HTTPS-доступ до container registry `registry.deckhouse.io` (установка также возможна и в [закрытом контуре](/ru/documentation/v1/deckhouse-faq.html#как-установить-deckhouse-из-стороннего-registry));
-   - SSH-доступ по ключу до узла, который будет **master-узлом** будущего кластера. 
-2. **Master-узел**.
+{%- if page.platform_type == "existing" %}
+   - SSH-доступ по ключу до **master-узла** существующего кластера.
+{%- else %} 
+   - SSH-доступ по ключу до узла, который будет **master-узлом** будущего кластера.
+{%- endif %}
+{% if page.platform_type == "existing" %}  
+1. **Существующий кластер Kubernetes**.
+ 
+   У вас уже должен быть работающий кластер Kubernetes, подходящий под условия [поддерживаемых версий Kubernetes и ОС](../../documentation/v1/supported_versions.html).
+    
+   Если в кластере нет Ingress-контроллера, добавьте в конфигурацию установки Deckhouse (файл config.yml на этапе 4 "Установка") включение модуля [ingress-nginx](../../documentation/v1/modules/402-ingress-nginx/) и создайте CR [IngressNginxController](../../documentation/v1/modules/402-ingress-nginx/cr.html#ingressnginxcontroller) с необходимыми параметрами после установки Deckhouse (см. пример на этапе 4 "Установка").
+  
+{%- else %}
+1. **Master-узел**.
  
-   Сервер {% if page.platform_type == 'baremetal' or page.platform_type == 'existing' %}(физический сервер или виртуальная машина){% else %}(виртуальная машина){% endif %}, который будет управляющим узлом (master-узлом) будущего кластера. В процессе установки, инсталлятор Deckhouse, запущенный на **персональном компьютере** (см. п.1), подключится к этому узлу по SSH, установит необходимые пакеты, настроит control plane Kubernetes и развернет Deckhouse. Установка **непосредственно с master-узла** в настоящий момент **не поддерживается**.
+   Сервер {% if page.platform_type == 'baremetal' or page.platform_type == 'existing' %}(физический сервер или виртуальная машина){% else %}(виртуальная машина){% endif %}, который будет управляющим узлом (master-узлом) будущего кластера.
+{%- endif %}
+
+   В процессе установки, инсталлятор Deckhouse, запущенный на **персональном компьютере** (см. п.1), подключится к **master-узлу** узлу по SSH, установит необходимые пакеты, настроит control plane Kubernetes и развернет Deckhouse. Установка **непосредственно с master-узла** в настоящий момент **не поддерживается**.
 
-   Требования: 
+   Требования{% if page.platform_type == 'existing' %} **к master-узлу**{%- endif %}: 
```

Comment:
There are no such requirements for nodes when we run just deckhouse-controller. Also there can be no master nodes, we should show how to deploy deckhouse in such conditions.

Reply:
Fixed